tests/utils.js

```diff
@@ -139,6 +139,7 @@ const deleteAll = (except) => {
     'branding',
     'partners',
     'settings',
+    'form:replace_user',
```

Comment:
FYI, this is kind of a hack that we need for now.  Without this, the form will be deleted by some of the existing tests.  I know that QA ran into the same problem in some of their tests and are working on a better solution, but for now this is the easiest way to  keep everything working.  We can change this to leverage QA's solution when we merge to `master`.